bgpd/bgp_advertise.h

```diff
@@ -62,7 +62,7 @@ struct bgp_advertise {
 	struct bgp_advertise_attr *baa;
 
 	/* BGP info.  */
-	struct bgp_info *binfo;
+	struct bgp_path_info *path;
```

Comment:
This should have been *bpInfo or *path_info instead of *path.

Reply:
The discussion we had for this bug is about "bgp_path" being not the right name and we concluded on bgp_path_info is the right one, because the information present in this structure is not just path, but also has some implementation details that are required for all the business logic operations around bgp_path. That was the argument to go towards bgp_path_info. On the same lines, the variable should indicate that this is path_info.  Not sure why do you think path is the right name here.

Reply:
no you concluded the name of the structure should be `struct bgp_path_info` . I disagree that a conclusion came forward that I cannot call variables path where appropriate.

Reply:
My impression is that, when we agree on Class/Struct names, same logic applies to names of Objects/Variables of such Class/Struct.

Reply:
@donaldsharp can you live with pinfo here?